lighthouse-core/runner.js

```diff
@@ -106,7 +106,12 @@ class Runner {
       }
 
       // Entering: conclusion of the lighthouse result object
-      const lighthouseVersion = require('../package.json').version;
+
+      // @ts-ignore - Needs json require() support
```

Comment:
@brendankenny do you have any tips for him here?

not sure why this would be be necessary now when it wasn't before 🤔

Reply:
I think this is just a bad merge or rebase. I just removed this `ts-ignore` in the last week or two